File: src/core/analyzer/iac/fixtures/cdk/aws-cdk-app.ts
import * as s3 from 'aws-cdk-lib/aws-s3';
import { Stack, StackProps } from 'aws-cdk-lib';
import { Construct } from 'constructs';

export class MyStack extends Stack {
  constructor(scope: Construct, id: string, props?: StackProps) {
    super(scope, id, props);

    const bucket = new s3.Bucket(this, 'LogsBucket', {
      versioned: true,
    });

    new s3.BucketPolicy(this, 'LogsPolicy', {
      bucket: bucket,
    });
  }
}

File: src/core/analyzer/iac/fixtures/cdk/aws_cdk_app.py
from aws_cdk import aws_s3 as s3
from constructs import Construct


class MyStack:
    def __init__(self, scope: Construct, id: str):
        bucket = s3.Bucket(self, "DataBucket", versioned=True)
        s3.BucketPolicy(self, "DataPolicy", bucket=bucket)
